README.md

```diff
@@ -10,33 +12,68 @@
 [![Known Vulnerabilities](https://snyk.io/test/npm/@ice/store/badge.svg)](https://snyk.io/test/npm/@ice/store)
 [![David deps](https://img.shields.io/david/ice-lab/icestore.svg?style=flat-square)](https://david-dm.org/ice-lab/icestore)
 
-## 安装
+## Installation
 
 ```bash
 npm install @ice/store --save
 ```
 
-## 快速上手
+## Introduction
+
+icestore is a lightweight React state management library based on hooks. It has following core features:
+
+* Minimal API: only 3 apis, easy to pick up in 5 minutes
+* Predictable: unidirectional data flow just like Redux, only allow state mutation inside actions, make data flow easier to trace
+* Optimal performance: multiple store design and only trigger rerender when state changed
```

Comment:
Optimal performance: Allows multiple stores to be created. Rerendering only occurs when the state changes.